src/drivers/ocpoc_sbus_rc_in/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+############################################################################
```

Comment:
@davids5 I imagine it's probably generic enough to be used as a general driver, but I don't really have the capacity to test that out. I would hate for someone down the line to assume it works when it may not, thus producing some major headaches.